### ISSUE TITLE
Subtitles add-on disabled

### ISSUE THREAD

**Issue by joe-13** [linuxmint/xplayer#113]
```
 * Xplayer version (xplayer --version) --> 1.8.3
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) --> Mint 19 Cinnamon
```

**Issue**
I want to download subtitles via xplayer, the add-on is disabled in the menu.
Hovering with the mouse on it says: "the python add-on charger could not be found".

**Steps to reproduce**
Any time I open a video with xplayer.

**Expected behaviour**
Make the add-on clickable in the menu, and be able to download subtitles.

**Other information**

**Comment by muzena** [comment]
I have same problema when i upgrade from mint 18 to mint 19.
I forgot name of plugin, but you can start xplayer from terminal and look in terminal output for error and name of missing plugin.

**Comment by leigh123linux** [comment]
Fixed in 2.0.0
